WireUI/Sources/WireDesign/Colors/ColorTheme.swift

```diff
@@ -0,0 +1,145 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UIKit
+
+public enum ColorTheme {
+
+    public enum Base {
+
+        public static let primary = UIColor(light: .blue500Light, dark: .blue500Dark)
+        public static let onPrimary = UIColor(light: .white, dark: .black)
+
+        public static let primaryVariant = UIColor(light: .blue50Light, dark: .blue800Dark)
+        public static let onPrimaryVariant = UIColor(light: .blue500Light, dark: .blue300Dark)
+
+        public static let error = UIColor(light: .red500Light, dark: .red500Dark)
+        public static let onError = UIColor(light: .white, dark: .black)
+
+        public static let warning = UIColor(light: .amber500Light, dark: .amber500Dark)
+        public static let onWarning = UIColor(light: .white, dark: .black)
+
+        public static let positive = UIColor(light: .green500Light, dark: .green500Dark)
+        public static let onSuccess = UIColor(light: .white, dark: .black)
+
+        public static let highlight = UIColor(light: .amber200Dark, dark: .amber300Dark)
+        public static let onHighlight = UIColor(light: .black, dark: .black)
+
+        public static let secondaryText = UIColor(light: .gray70, dark: .gray60)
+    }
+
+    public enum Backgrounds {
+
+        public static let background = UIColor(light: .gray20, dark: .gray100)
+        public static let onBackground = UIColor(light: .black, dark: .white)
+
+        public static let backgroundVariant = UIColor(light: .gray10, dark: .gray95)
+        public static let onBackgroundVariant = UIColor(light: .black, dark: .white)
+
+        public static let surface = UIColor(light: .white, dark: .gray95)
+        public static let onSurface = UIColor(light: .black, dark: .white)
+
+        public static let surfaceVariant = UIColor(light: .white, dark: .gray90)
+        public static let onSurfaceVariant = UIColor(light: .black, dark: .white)
+
+        public static let inverted = UIColor(light: .black, dark: .white)
+        public static let onInverted = UIColor(light: .white, dark: .black)
```

Comment:
It's not clear to me when to reach for these colors. Are these names that are defined in the Wire Design system on Figma? Or do they have some intended uses? If so, perhaps some comments may help guide us to use them correctly.

Reply:
![Screenshot 2024-06-28 at 12 08 38](https://github.com/wireapp/wire-ios/assets/15628584/2408fdc4-87e7-4ce0-918a-cef6a670b77b)

The colors are mentioned in the design.